extensions-core/azure-extensions/src/main/java/org/apache/druid/storage/azure/AzureTaskLogs.java

```diff
@@ -151,14 +167,36 @@ private String getTaskReportsKey(String taskid)
   }
 
   @Override
-  public void killAll()
+  public void killAll() throws IOException
   {
-    throw new UnsupportedOperationException("not implemented");
+    log.info("Deleting all task logs from Azure storage location [bucket: %s    prefix: %s].",
+             config.getContainer(), config.getPrefix()
+    );
```

Comment:
same comment about 1 line per arg formatting if spans multiple lines
```suggestion
    log.info(
        "Deleting all task logs from Azure storage location [bucket: %s    prefix: %s].",
        config.getContainer(),
        config.getPrefix()
    );
```

Reply:
done